src/utils/rtl.test.tsx

```diff
@@ -0,0 +1,282 @@
+import { MultiSelectField, NestedOption, SelectField, TreeSelectField } from "src/inputs";
+import { HasIdAndName } from "src/types";
+import { getOptions, getSelected, render, select, selectAndWait } from "src/utils/rtl";
+
+describe("rtl", () => {
+  it("can use select helpers and select an option via value on SelectField", async () => {
+    const onSelect = jest.fn();
+    // Given the SelectField
+    const r = await render(
+      <SelectField
+        label="Number"
+        value={undefined}
+        onSelect={onSelect}
+        options={[
+          { id: "1", name: "One" },
+          { id: "2", name: "Two" },
+          { id: "3", name: "Three" },
+        ]}
+      />,
+    );
+    // Then the getOptions helper returns the correct options
+    expect(getOptions(r.number)).toEqual(["One", "Two", "Three"]);
+
+    // When selecting an option
+    select(r.number, "2");
```

Comment:
Question: Fwiw what do you think about "Two" instead of "2"? I think that's how our current DSL works, for native selects, so will be less code to change/update to use the id instead? Or am I wrong about that...

Reply:
You can do both! If you want to select by the label or value of the element you can. 
But you're right, selecting by the label is how we currently do it. But I kind of liked giving the user the option of choosing how they'd want to do it.

Reply:
Ah nice! Agreed supporting both sgtm.